src/overlays/actors/ovl_Bg_Bdan_Objects/z_bg_bdan_objects.h

```diff
@@ -6,12 +6,22 @@
 
 struct BgBdanObjects;
 
+typedef enum {
+    BDAN_OBJECT_TYPE_BIG_OCTO_PLATFORM = 0,
+    BDAN_OBJECT_TYPE_SMALL_AUTO_ELEVATOR = 1,
+    BDAN_OBJECT_TYPE_WATERBOX_HEIGHT_CHANGER = 2,
+    BDAN_OBJECT_TYPE_FALLING_PLATFORM = 3
```

Comment:
you used `JABU_OBJECTS_` as a prefix for the get/set property enums, it'd probably be a good idea to use the same as here

so `JABU_OBJECTS` or `BDAN_OBJECTS` (mind the `S`, following the actor's debug name, though that may be up to discussion)

Reply:
hmm I guess I should stick with `BDAN_OBJECTS` for now? seems more correct, maybe some eventual rename of the actor in the future to friendly common names will then go to JABU.

Reply:
imo its fine to use friendly names for all enums in an actor, but think everyone has their own opinions/preferences on this

Reply:
changed to friendlies in aefde5c